docs/install/tutorials/rpi.rst

```diff
@@ -163,14 +153,34 @@ Copy and paste the following text, then press :guilabel:`CTRL` + :guilabel:`X` t
   # Let the Raspberry Pi resolve to all DNS queries
   address=/#/192.168.4.1
 
-.. warning::
+  interface=wlan0
+
+    # Gateway + DNS server
+    dhcp-option=3,192.168.4.1
+    dhcp-option=6,192.168.4.1
+
+    # specify the range of IPs that will be handed out
+    dhcp-range=192.168.4.2,192.168.4.200,255.255.255.0,24h
 
-  These settings override the possibility to connect to an online source using the Wi-Fi. It is still possible to connect to the internet **through the cabled network**, however you will need to configure a DNS server manually every time you reboot the device. Put the IP of your DNS provider in ``/etc/resolve.conf``. If you don't know it, you can use Google's OpenDNS address ``8.8.8.8`` as in this example:
```

Comment:
The line below (`DNSMASQ_EXCEPT=lo`) makes this unnecessary -- it no longer stomps on /etc/resolv.conf! :fireworks:

Reply:
Interesting, thanks so much!